app/src/main/java/io/homeassistant/companion/android/settings/vehicle/ManageAndroidAutoViewModel.kt

```diff
@@ -45,7 +45,14 @@ class ManageAndroidAutoViewModel @Inject constructor(
                 async {
                     entities[it.id] = try {
                         serverManager.integrationRepository(it.id).getEntities().orEmpty()
-                            .filter { it.domain in MainVehicleScreen.SUPPORTED_DOMAINS }
+                            .filter {
+                                it.domain in MainVehicleScreen.SUPPORTED_DOMAINS ||
+                                    (
+                                        it.domain in MainVehicleScreen.MAP_DOMAINS &&
+                                            ((it.attributes as Map<*, *>)["latitude"] as Double? != null) &&

```

Comment:
These checks should use `as? Double` to be consistent with the car service implementation and prevent a crash in case `latitude` exists but it isn't a number.

Reply:
done thanks for calling that out :)